JobScheduler.js

```diff
@@ -183,6 +194,18 @@ class JobScheduler {
     });
   }
 
+  isServiceFabrikUp() {
+    return serviceFabrikClient.getInfo()
+      .then(info => {
+        logger.info('SF Status :', info);
+        if (info.db_status === CONST.DB.STATE.CONNECTED) {
+          logger.info('SF is connected to DB, can go ahead with starting scheduler.');
```

Comment:
This logger statement, you can put in caller. As based on connected status, what to do, let caller decide. Just a thought !

Reply:
Unnecessary logger. removing it.